code-of-conduct.md

```diff
@@ -72,16 +72,18 @@ Conduct may be permanently removed from the project team.
 
 For incidents occuring in the Kubernetes community, contact the [Kubernetes Code of Conduct Committee](https://git.k8s.io/community/committee-code-of-conduct) via <conduct@kubernetes.io>. You can expect a response within three business days.
 
-For other projects, please contact the CNCF staff via <conduct@cncf.io>. You can expect a response within three business days. 
+For other projects, or for incidents that are project-agnostic or impact multiple CNCF projects, please contact the [CNCF Code of Conduct Committee](https://www.cncf.io/conduct/committee/) via conduct@cncf.io.  Alternatively, you can contact any of the individual members of the [CNCF Code of Conduct Committee](https://www.cncf.io/conduct/committee/) to submit your report. For more detailed instructions on how to submit a report, including how to submit a report anonymously, please see our Incident Resolution Procedures. You can expect a response within three business days.
```

Comment:
need a link to `Incident Resolution Procedures`

Reply:
https://www.cncf.io/conduct/procedures/